### PR TITLE
added wishlist functionality

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Cart Functionality along with it's route.
-> Used Local Storage so that even after refreshing the page..user data persist .
-> Also handled the responsiveness.

### CODE REVIEW

File: src/Components/Mobile_filter/Mobile_filter.jsx [hunk 1]
```diff
@@ -104,17 +105,15 @@ export const Mobile_filter = () => {
...
-                }}
-              >
+            {ratings.map((ele, index) => (
+              <label key={index} className="radio flex-center-row gap-0-5">
                 <input
+                  onChange={() => {
+                    setFilterState({
+                      type: "rating",
+                      paylod: ele,
+                    });
```
Comment (Richa-rani02) [inline src/Components/Mobile_filter/Mobile_filter.jsx]: suggestion: better to define type in capital letter .just a good practice

**Review by Richa-rani02 — APPROVED** [review]

**Review by AmishMishra11 — APPROVED** [review]
Great work just while defining type you should use capital letters as its a good practice

**Review by Mohitkumar33 — APPROVED** [review]
Nice

File: src/Components/Mobile_filter/Mobile_filter.jsx (hunk 1 above)
Comment (ashishnnnnn) [reply src/Components/Mobile_filter/Mobile_filter.jsx]: Done ..

**Comment by ashishnnnnn** [comment]
Done the changes.

**Pull request merged**